ci/kokoro/docker/build.sh

```diff
@@ -204,9 +206,7 @@ docker_flags=(
     "--env" "CXX=${CXX}"
     "--env" "CC=${CC}"
 
-    # The number of CPUs, probably should be removed, the scripts can detect
-    # this themselves in Kokoro (it was a problem on Travis).
-    "--env" "NCPU=${NCPU:-4}"
+    "--env" "${NCPU}"
```

Comment:
Drive by: You still want the "NCPU=" part, right?

Reply:
Thanks, you're correct. I'm closing this PR though because I remember why I didn't do this earlier. `NCPU` is only used in one Dockerfile.